packages/oraidex-common/src/network.ts

```diff
@@ -940,6 +940,7 @@ export const network: CustomChainInfo & NetworkConfig = {
 
 // exclude kawaiverse subnet and other special evm that has different cointype
 export const evmChains = chainInfos.filter((c) => c.networkType === "evm");
+export const cosmosChains = chainInfos.filter((c) => c.networkType === "cosmos");
```

Comment:
I'd favour if this network type being constanted as well. But we can review this later.